src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet
+{ 
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
```

Comment:
Ah, I remember now. It's the approach neither of us liked :smile:. Why are we doing it again? Does this require changes in our telemetry reports?